style.css

```diff
@@ -214,12 +214,14 @@ body {
 .color_for_link {
     color: aliceblue;
     text-decoration: none;
-    font-size: 0.8rem;
+    font-size: 1.5rem;
+    margin-left: 5px;
 }
 
 .color-icon {
     color: rgb(30, 119, 202);
     margin-left: 20px;
+    font-size: 30px;
```

Comment:
It's better to keep one strategy of sizing . You should use px or rem . 
Some font-size has px , another - rem . You need to keep something one.

Reply:
did